AutoGVP/01-annotate_variants_CAVATICA_input.R

```diff
@@ -97,13 +98,16 @@ Sys.setenv("VROOM_CONNECTION_SIZE" = 131072 * 2)
 ## function for filtering gnomAD, variant af and depth filtering 
 gnomad_filtering <- function(clinvar_vcf) 
 {
-  gnomad_var_to_match <- paste0(filter_gnomad_var,"\\=(0\\.\\d+)\\;")
+  gnomad_var_to_match <- paste0(gnomad_variable,"\\=(0\\.\\d+)\\;")
+
   clinvar_vcf <- clinvar_vcf %>% 
                   dplyr::mutate(
-                    variant_depth = if_else( as.integer( str_match(INFO, "DP\\=(\\d+)")[, 2])  > filter_variant_depth, "PASS","FAIL"),
-                    gnomad_af     = if_else( as.numeric( str_match(INFO, gnomad_var_to_match)[,2])  > filter_variant_af, "PASS","FAIL"),
-                    variant_af    = if_else(as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3]) / ( (as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,2]) ) + as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3] )) > filter_variant_af, "PASS", "FAIL")
+                    variant_depth = as.integer( str_match(INFO, "DP\\=(\\d+)")[, 2]),
+                    gnomad_af     = as.numeric( str_match(INFO, gnomad_var_to_match)[,2]),
+                    #variant_af    = as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3]) / ( (as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,2]) ) + as.integer(str_match(Sample, ":(\\d+)\\,(\\d+)") [,3] )) 
                     )
+  clinvar_vcf <- clinvar_vcf %>% dplyr::filter(variant_depth > filter_variant_depth & gnomad_af > filter_gnomad_af) 
```

Comment:
```suggestion
  clinvar_vcf <- clinvar_vcf %>% 
  dplyr::filter(variant_depth >= filter_variant_depth & gnomad_af >= filter_gnomad_af) 
```
can you move the filter to the second line for readability, and I think we may also want to use this as min depth/AF